monai/transforms/compose.py

```diff
@@ -254,7 +254,7 @@ def __call__(self, data):
     """
 
     def __init__(self, keys: KeysCollection) -> None:
-        self.keys: Tuple[Any, ...] = ensure_tuple(keys)
```

Comment:
Since we know we're assigning a tuple for sure we could have left this type hint alone. The change to Sequence elsewhere is mainly for arguments which can deal with tuple or list since we're going to convert them later or pass them somewhere that accepts any sort of sequence.

Reply:
updated in the latest commit.
Thanks.